File: asi_HossameldinYounes_tasks/asi_HossameldinYounes_CProgrammingForStudents/asi_HossameldinYounes_GuessingGame.c
#include <stdio.h>

int main() {
    
    int secretnumber=5;
    int guess,guesscount = 0;
    int guesslimit = 2;
    int outofguess = 0;
    
    printf("Enter your guess: ");
    scanf("%d",&guess);
    
    if(guess == secretnumber){
            printf("You win");
    }
    
    while(guess != secretnumber){
        if(guesscount < guesslimit){
            printf("Enter your guess: ");
            scanf("%d",&guess);
            guesscount++;
        }else{
            printf("You are out of guesses");
            break;
        }
    }
    return 0;
}

File: asi_HossameldinYounes_tasks/asi_HossameldinYounes_PythonStudent/asi_HossameldinYounes_GuessingGame.py
word = "hossam"
guessCount = 1


guess = input("Enter your guess: ")

if guess == word:
    print("You Win!")

while guess != word:
    if guessCount >= 3:
        print("You lose! ")
        break
    else:
        guess = input("Enter your guess: ")
        guessCount = guessCount + 1

if guess == word:
    print("You Win!")